packages/client/components/modals/jobs/JobPost.tsx

```diff
@@ -86,7 +86,7 @@ const JobModalContent = ({ description, technologyTags }: any) => {
     <Grid
       gridTemplateColumns={{
         lg: 'repeat(2, 1fr)',
-        sm: 'repeat(1, minmax(150px, 1fr))',
```

Comment:
The smallest definition _has to be_ `base` and not `sm`. Here we set values for `base` and `lg`, which means breakpoints >= `lg` use the `lg` value and breakpoints < `lg` use the `base` value. (Type 1)